packages/js-client/src/context.ts

```diff
@@ -0,0 +1,71 @@
+import { SimpleStorageContextParams } from './types';
+import { Context, ContextCore, ContextState } from '@aragon/sdk-client-common';
+
+// TODO
+// import from @aragon/sdk-client-common
+type OverriddenState = {
+  daoFactoryAddress: boolean;
+  ensRegistryAddress: boolean;
+  gasFeeEstimationFactor: boolean;
+  ipfsNodes: boolean;
+  graphqlNodes: boolean;
+};
+
+type SimpleStorageContextState = ContextState & {
+  // extend the Context state with a new state for storing
+  // the new parameters
+  myParam: string;
+};
+
+type SimpleStorageOverridenState = OverriddenState & {
+  myParam: boolean;
+};
```

Comment:
Why is this needed? The top section seems to have more boilerplate than necessary

Reply:
This is just an examplo of how the extend the `OverridenState`, the `OverridenState` type itself should be imported from the `client-common` package

Reply:
My point is that it looks like we are defining the same type twice, in different ways